Sources/SnowManLogic/GameLogic/Game.swift

```diff
@@ -0,0 +1,45 @@
+public class Game {
+	var word: String
+	var guessManager: GuessManager
+	var view: View
+	var guess: String
+
+	public init(word: String, guessManager: GuessManager, view: View) {
+		self.word = word
+		self.guessManager = guessManager
+		self.view = view
+		guess = ""
+	}
+
+	public func playGame() {
+		while (gameIsOver() == false) {
+			playerTurn()
+		}
+		view.displayResults(word, winner: isWinner())
+	}
+
+	public func gameIsOver() -> Bool {
+		return guessManager.hasNoGuessesRemaining() || isWinner()
+	}
+
+	public func isWinner() -> Bool {
+		let letters = separateLetters()
+		return guessManager.determineUnguessedLetters(letters).isEmpty || guessManager.correctlyGuessedFullWord(word, guess: guess)
+	}
+
+	private func playerTurn() {
+		view.assignBlanks(word, correctGuesses: guessManager.correctGuesses)
+		guess = view.receiveGuess(view.getGuessFromUser)
```

Comment:
Why are you passing a method that belongs to an instance to a method that belongs to the same instance?